File: server-scala-play-server/app/model/ProductResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ProductResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ProductResponse(
  brand: Option[String],
  productType: Option[String],
  condition: Option[String],
  isbn: Option[String],
  asin: Option[String],
  catalogNumbers: Option[List[String]],
  department: Option[String],
  features: Option[List[String]],
  minimumPrice: Option[Double],
  width: Option[Double],
  height: Option[Double],
  depth: Option[Double],
  weight: Option[Double],
  unit: Option[String],
  studio: Option[String],
  parentalRating: Option[String],
  publishDate: Option[Long],
  availabilityDate: Option[Long],
  size: Option[SizeGroupResponse]
)

object ProductResponse {
  implicit lazy val productResponseJsonFormat: Format[ProductResponse] = Json.format[ProductResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ProductResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ProductResponse 
{
    public string Brand { get; set; }
    public string ProductType { get; set; }
    public string Condition { get; set; }
    public string Isbn { get; set; }
    public string Asin { get; set; }
    public List<string> CatalogNumbers { get; set; }
    public string Department { get; set; }
    public List<string> Features { get; set; }
    public double MinimumPrice { get; set; }
    public double Width { get; set; }
    public double Height { get; set; }
    public double Depth { get; set; }
    public double Weight { get; set; }
    public string Unit { get; set; }
    public string Studio { get; set; }
    public string ParentalRating { get; set; }
    public long PublishDate { get; set; }
    public long AvailabilityDate { get; set; }
    public SizeGroupResponse Size { get; set; }
}


